src/wymeditor/parser/xhtml-validator.js

```diff
@@ -21,13 +21,21 @@ WYMeditor.XhtmlValidator = {
             "attributes":[
             "class",
             "id",
-            "style",
             "title",
             "accesskey",
             "tabindex",
             "/^data-.*/"
             ]
         },
+        "styleAttr":
+        {
+            "except":[
+            "img"
+            ],
+            "attributes":[
+            "style"
+            ]
+        },
```

Comment:
And then I added the `style` attribute to be valid on all elements except `img`.